src/init/clickhouse_exporter.rs

```diff
@@ -4,61 +4,76 @@ use clap::{Args, ValueEnum};
 #[derive(Debug, Clone, Args)]
 pub struct ClickhouseExporterArgs {
     /// Clickhouse Exporter endpoint
-    #[arg(long, env = "ROTEL_CLICKHOUSE_EXPORTER_ENDPOINT")]
-    pub clickhouse_exporter_endpoint: Option<String>,
+    #[arg(
+        id("CLICKHOUSE_ENDPOINT"),
```

Comment:
Clap requires the parameter names to be unique, because we also have endpoint in OTLP exporter we have to use this id() function to make them unique.